File: backend/src/main/java/com/romeo/VetLink/appointment/service/dtos/AppointmentDTO.java
package com.romeo.VetLink.appointment.service.dtos;


import com.romeo.VetLink.patients.service.dtos.PatientDTO;
import lombok.Data;

@Data
public class AppointmentDTO {
    private Integer id;

    private String job;
    private String eventLength;
    private Boolean unscheduled;

    private PatientDTO patientDTO;
    private ConfirmedScheduleDTO confirmedScheduleDTO;
}

File: frontend/src/app/appointment-section/domain/appointment-dto.ts
import {PatientDto} from "../../patient-section/domain/patient-dto";
import {ConfirmedScheduleDto} from "./confirmed-schedule-dto";

export class AppointmentDto {
  id: number | null = null;
  patientDTO: PatientDto | null = null
  job: string | null = null;
  color: string | null = null;
  eventLength: number | null = null;
  unscheduled: boolean = false;
  confirmedScheduleDTO: ConfirmedScheduleDto | null = null

}
